recipes/paradict/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "paradict" %}
+{% set version = "0.0.3" %}
```

Comment:
Looks like the latest version is 0.0.5. Would it make sense to use that or is 0.0.3 needed for some reason?

Reply:
Probably it updated since I prepared this recipe. We might as well merge this now and let the version update at the feedstock?